Undefined symbol "_OBJC_CLASS_$_GIDSignIn"

<!-- DO NOT DELETE
validate_template=true
template_path=.github/ISSUE_TEMPLATE/bug_report.md
-->

### [REQUIRED] Step 1: Describe your environment

  * Xcode version: 12.4 (12D4e)
  * Firebase SDK version: 7.5.1
  * Installation method: `Swift Package Manager`
  * Firebase Component: all but issue related to auth I believe

### [REQUIRED] Step 2: Describe the problem

#### Steps to reproduce:

I included Firebase using Swift Package Manager and upon compilation of my project I get the error:

Undefined symbols for architecture arm64:
  "_OBJC_CLASS_$_GIDSignIn", referenced from:
      objc-class-ref in VCReauthenticate.o
      objc-class-ref in AppDelegate.o
      objc-class-ref in VCLogin.o
      objc-class-ref in VCAccount.o
ld: symbol(s) not found for architecture arm64

--- a/ReleaseTooling/Sources/ZipBuilder/ResourcesManager.swift
+++ b/ReleaseTooling/Sources/ZipBuilder/ResourcesManager.swift
@@ -241,6 +241,11 @@ extension ResourcesManager {
         continue
       }
 
+      if fullPath.lastPathComponent.hasSuffix("bundle") {
+        // It's already a bundle, so no need to create one.
+        continue
+      }
+
       // It's a folder. Generate the name and location based on the folder name.
       let name = fullPath.lastPathComponent + ".bundle"
       let location = dir.appendingPathComponent(name)

